dbt_project.yml

```diff
@@ -1312,6 +1312,10 @@ models:
       ethereum:
         +schema: tokemak_ethereum
 
+    jupiter:
+      +schema: jupiter
+      solana:
+        +schema: jupiter_solana
```

Comment:
let's remove from project file and just apply schema in models directly, that's future state direction